packages/react-components/src/components/VideoTile.tsx

```diff
@@ -295,6 +302,19 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
                   <Icon iconName="VideoTileMicOff" />
                 </Stack>
               )}
+              {
+                /* @conditional-compile-remove(pinned-participants) */
+                isPinned && (
+                  <Stack className={mergeStyles(iconContainerStyle)}>
+                    <Icon
+                      iconName="PinnedSolid"
```

Comment:
We need to make this icon customizable by Contoso

Reply:
done